menu.js

```diff
@@ -7,7 +7,7 @@ module.exports = function(electronApp, menuState) {
     enabled: function() {
 
       // only enabled for BPMN diagrams
-      return menuState.bpmn && menuState.inactiveInput;
+      return menuState.bpmn && menuState.platform == 'platform' && menuState.inactiveInput;
```

Comment:
This is not backwards compatible.

Consider changing the check to `menuState.platform === 'cloud'`.

Reply:
Ah good point I hadn't considered the backwards compatibility requirements of plugins. Let me have a look on how I could change this.